regression/contracts/assigns_enforce_05/test.desc

```diff
@@ -1,9 +1,10 @@
 CORE
 main.c
 --enforce-all-contracts
-^EXIT=10$
+^EXIT=0$
 ^SIGNAL=0$
-^VERIFICATION FAILED$
+^VERIFICATION SUCCESSFUL$
```

Comment:
Is there any way we could add some additional regexes to make sure that contracts are processed? The enforce flag should be generating some additional assertions, so can we match some regex on them?

The idea here is to fail this test (instead of vacuously passing) if somehow `--enforce-all-contracts` flag is a no-op.

Reply:
Implemented as suggested.